.github/workflows/windows.yml

```diff
@@ -66,15 +65,14 @@ jobs:
 
       - uses: actions/setup-python@v4
         with:
-          python-version: '3.11'
+          python-version: ${{ env.PYTHON_VERSION }}
 
       - name: Install python dependencies
         run: |
           # For Python API
           python3 -m pip install Scons
           python3 -m pip install -r ${{ env.OPENVINO_REPO }}/src/bindings/python/wheel/requirements-dev.txt
           python3 -m pip install -r ${{ env.OPENVINO_REPO }}/src/bindings/python/requirements.txt
```

Comment:
`${{ env.OPENVINO_REPO }}/src/bindings/python/requirements.txt` contains product requirements, which are not required for build

But we need to add requirement for legacy OpenVINO API build https://github.com/openvinotoolkit/openvino/blob/master/.github/workflows/linux.yml#L104